### DISCUSSION TITLE
zotero文献阅读可以同时打开两篇不同的文献对比阅读吗

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#105] [求助 / Q&A] [answered]
zotero文献阅读可以同时打开两篇不同的文献对比阅读吗，我知道同一篇文献的横向竖向分屏阅读，不同文献可以吗

**Comment by northword** [comment]
使用单独窗口打开文献

![image](https://github.com/zotero-chinese/wiki/assets/44738481/4035df99-210f-4a80-b89b-8d3583ad12ae)


![image](https://github.com/zotero-chinese/wiki/assets/44738481/85a963ef-a0f5-498e-a70a-31159b0a8040)

**Comment by LHT-Liu** [comment]
请问这个是7特有的吗，我在6里面好像没找到这个

**Comment by northword** [comment]
记不清了，没有的话就是7特有的吧

**Comment by Jerome-Cong** [comment, marked as answer]
6里面已经打开的pdf可以通过右键移动标签到新窗口来多开

**Reply by yuanozi** [reply]
请问有快捷键吗 比如按着ctrl点可以直接从新窗口打开？

**Comment by LHT-Liu** [comment]
> 6里面已经打开的pdf可以通过右键移动标签到新窗口来多开

好的好的，感谢您

**Comment by liudomain** [comment]
zotero7 在上面的基础上还需要设置
点击要分开的Zotero文献条目-移动标签-移动到新窗口，适用于新建笔记和文献对比阅读

<img width="936" alt="微信图片_20240716112604" src="https://github.com/user-attachments/assets/5a598e3a-1c59-4df5-bd26-4e8c8f65edea">
<img width="923" alt="微信图片_20240716112802" src="https://github.com/user-attachments/assets/92d05100-171e-48c8-bbd4-17a739bb5fb6">

**Comment by weikang123340** [comment]
感谢

**Reply by weikang123340** [reply]
这种方法--Zotero文献条目-移动标签-移动到新窗口，适用于新建笔记和文献对比阅读可以的

**Comment by AllenZF** [comment]
能在原窗口实现吗，感觉新打开一个窗口不优雅。
